content/pages/blogposts/2024-10-14-community-over-code-na-birds-of-a-feather.md

```diff
@@ -0,0 +1,11 @@
+Title: Community Over Code NA 2024 Apache Lucene/Solr Birds Of A Feather
+category: solr/blogposts
+summary: This blog post is a summary of the Apache Lucene/Solr Birds of a Feather from Community Over Code NA 2024 in Denver. Written by Eric Pugh and Stefan Vodita.
+slug: community-over-code-na-2024-birds-of-a-feather
+URL: blogposts/community-over-code-na-2024-birds-of-a-feather.html
+save_as: blogposts/community-over-code-na-2024-birds-of-a-feather.html
+
+# Community Over Code NA 2024 Apache Lucene/Solr Birds Of A Feather
+Dive into the insights and key takeaways from the Apache Lucene/Solr Birds of a Feather (BoF) session at Community Over Code EU 2024, held in the mile high city of Denver, Colorado. This session was specially organized to gather feedback from the community, fostering a collaborative environment where developers, users, and enthusiasts could share their experiences and insights. 
```

Comment:
A few suggestions:

- [0] maybe put "Birds of a Feather" in double-quotes?
- [-1] Drop the "EU"...I imagine this is a copy/paste error?